### DISCUSSION TITLE
Integrazione tado

### DISCUSSION THREAD

**Discussion by Matty1184** [indomus/forum#3317] [GENERALE - Discussioni generiche]
ciao a tutti,
ho installato un termostatato tado x con il suo bridge, volevo integrarlo in HA in modalità cloud.
nell'integrazione ho inserito le mie credenziali ma putroppo non visualizza il dispositivo. mi compaiono delle entità riferite solo al meteo esterno dell'account.
suggerimenti?

**Comment by KC-inDomus** [comment]
devi e puoi usare solo matter.
